app/steps/ui/payment/breakdown/index.js

```diff
@@ -66,85 +67,67 @@ class PaymentBreakdown extends Step {
         const authorise = new Authorise(config.services.idam.s2s_url, ctx.sessionID);
         const serviceAuthResult = yield authorise.post();
         if (serviceAuthResult.name === 'Error') {
-            const options = {};
-            options.redirect = true;
-            options.url = `${this.steps.PaymentBreakdown.constructor.getUrl()}?status=failure`;
-            formdata.paymentPending = 'unknown';
-            return options;
+            logger.info(`serviceAuthResult Error = ${serviceAuthResult}`);
+            const keyword = 'failure';
+            errors.push(FieldError('authorisation', keyword, this.resourcePath, ctx));
+            return [ctx, errors];
         }
+
+        const [result, submissionErrors] = yield this.sendToSubmitService(ctx, errors, formdata, ctx.total);
```

Comment:
As discussed can you wrap the sendToSubmit code including the setting the ccdCase.id and ccdCase.state with an if statement. If the ccdCase.id already exists in the formdata then skip this section.